qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -514,6 +514,12 @@ def can_go_back(self):
     def can_go_forward(self):
         return self._history.canGoForward()
 
+    def back_items(self, max_items: int = 99999):
+        return self._history.backItems(max_items)
```

Comment:
I'm not sure we should expose `max_items` at all - what about just passing `self._history.count()` there?